app_legacy/Helpers/database/subscription.php

```diff
@@ -9,22 +9,19 @@
  */
 function updateSubscription(string $subjectType, int $subjectID, int $userID, bool $state): bool
 {
-    sanitize_sql_inputs($subjectType, $subjectID, $userID, $state);
-    settype($state, 'integer');
+    sanitize_sql_inputs($subjectType);
+    $state = (int) $state;
```

Comment:
I think this line can be eliminated because of the logic on line 15.

Reply:
Ah yes. I updated most of the bool to int casts to work the same.
This works just fine:
```php
echo (int) true;
// 1
```